alb.tf

```diff
@@ -94,9 +102,10 @@ resource "null_resource" "alb_target_group" {
     command = <<CMD
 echo "\
 export ALB_TARGET_GROUP=${module.alb.default_alb_target_group} 
-export ALB_ROLE=/ecs/acc_ecs_lb_role
+export ALB_ROLE=/ecs/dev_ecs_lb_role
```

Comment:
now that we have a working build, can you please add the ${var.environment} variable here, instead of hardcoding dev.